### PR TITLE
Transport.py refactor

### PR DESCRIPTION
**Pull request by Ruzihm**
Saw that Transport.py had a bunch of issues and complexity on codacy ([see here](https://app.codacy.com/gh/unitystation/unitystation/file/49474191654/issues/source?bid=20699314&fileBranchId=20699314)). This is a take at reducing some of that.

I am not 100% sure on how to test this to make sure I didn't break anything in the refactor ~~so please don't merge this just yet.~~

### CODE REVIEW

File: Tools/python DM botany to json/Transport.py [hunk 1]
```diff
@@ -1,68 +1,137 @@
...
+        if inheritance:
+            inherent[tl_naem][trait_name] = trait
+
+        trait = F"\t{trait}\n"
+        full = full + trait
+
+    return full,trait
+
+
+def burer_piopes(burer, text_buffer, inherent):
```
Comment (Bod9001) [inline Tools/python DM botany to json/Transport.py:124]: Codacy found an issue: [burer_piopes is too complex (25) (MC0001)](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6388395)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 41
- Added 1
           

Complexity decreasing per file
==============================
+ Tools/python DM botany to json/Transport.py  -74
         

Clones removed
==============
+ Tools/python DM botany to json/Transport.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6388395&bid=20716944)

**Comment by corp-0** [comment]
we don't really use python in the project, those are mostly system prototypes and little tools. This one in particular was used to import data from DM to the game.

**Comment by Ruzihm** [comment]
Ah, that's good to hear. Sounds like it's fine to merge. I figure if its worth keeping in develop it's worth making readable ;)

**Comment by NoooneyDude** [comment]
Yeah and the `Codacy` rating has been bouncing between `A` and `B`; this'll set keep us at `A` for a good while ; )

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 41
           

Complexity decreasing per file
==============================
+ Tools/python DM botany to json/Transport.py  -89
         

Clones removed
==============
+ Tools/python DM botany to json/Transport.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6388395&bid=20716944)

**Comment by Ruzihm** [comment]
That's good for now. I hear @Bod9001 may know how to test it.

Either way, since Transport.py is not actively used I suppose merging it for now wont hurt ;)

**Comment by corp-0** [comment]
yup, merging

**Pull request merged**